src/Rendering/Renderer.cpp

```diff
@@ -147,6 +151,12 @@ void Renderer::PostDraw(const flecs::iter& iter, int index)
 	rlImGuiEnd();
 	EndDrawing();
 }
+void Renderer::UpdateCoinsCounter(flecs::entity entity, const CoinsComponent& coinsComponent)
+{
+	char coinText[50];
```

Comment:
In C++ we'll prefer to rely on `std::string`, it brings more safety and more convenience, so we'd see something like that instead:
```cpp
	std::string coinText = "Coins :" + std::to_string(coinsComponent.mCoinAmount);
	DrawText(coinText.c_str(), 10, HEIGHT-20, 20, RAYWHITE);
```

Additionally, for building strings with much more elements, we'll prefer to a use `stringstream`: https://en.cppreference.com/w/cpp/io/basic_ostream/operator_ltlt